app/src/main/res/layout/activity_bill.xml

```diff
@@ -0,0 +1,282 @@
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8"
+                            car_view:cardElevation="@dimen/dp1">
+
+                            <LinearLayout
+                                android:layout_width="match_parent"
+                                android:layout_height="match_parent"
+                                android:orientation="vertical"
+                                android:padding="@dimen/dp_5">
+
+                                <com.toptoche.searchablespinnerlibrary.SearchableSpinner
+                                    android:layout_width="match_parent"
+                                    android:layout_height="wrap_content"
+                                    android:scrollbarSize="@dimen/dp_20"
+                                    bind:hintText="@string/title_stylist"
+                                    bind:searchableSpinnerAdapter="@{viewModel.stylistAdapter}"
+                                    bind:stylist="@={viewModel.stylist}"/>
+                            </LinearLayout>
+                        </android.support.v7.widget.CardView>
+
+                        <android.support.design.widget.TextInputLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8">
+
+                            <EditText
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:hint="@string/title_price"
+                                android:inputType="number"
+                                android:text="@={viewModel.price}"
+                                android:textSize="@dimen/sp_10"/>
+                        </android.support.design.widget.TextInputLayout>
+
+                        <android.support.design.widget.TextInputLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="wrap_content"
+                            android:paddingEnd="@dimen/dp_8"
+                            android:paddingStart="@dimen/dp_8">
+
+                            <EditText
+                                android:layout_width="match_parent"
+                                android:layout_height="wrap_content"
+                                android:hint="@string/title_number"
+                                android:inputType="number"
+                                android:text="@={viewModel.qty}"
+                                android:textSize="@dimen/sp_10"/>
+                        </android.support.design.widget.TextInputLayout>
+                    </LinearLayout>
+
+                    <LinearLayout
```

Comment:
WARNING: [UseCompoundDrawables] This tag and its children can be replaced by one `<TextView/>` and a compound drawable
 A `LinearLayout` which contains an `ImageView` and a `TextView` can be more efficiently handled as a compound drawable (a single TextView, using the `drawableTop`, `drawableLeft`, `drawableRight` and/or `drawableBottom` attributes to draw one or more images adjacent to the text).

If the two widgets are offset from each other with margins, this can be replaced with a `drawablePadding` attribute.

There's a lint quickfix to perform this conversion in the Eclipse plugin.